openvdb_points/tools/AttributeArray.h

```diff
@@ -542,11 +563,12 @@ class TypedAttributeArray: public AttributeArray
 
 /// AttributeHandles provide access to specific TypedAttributeArray methods without needing
 /// to know the compression codec, however these methods also incur the cost of a function pointer
-template <typename T>
+template <typename T, bool Strided = false, bool Interleaved = false>
```

Comment:
Why does the handle need to be templated by the user for Strided and Interleaved?